index.bs

```diff
@@ -264,13 +264,13 @@ aligned (8) class AV1CodecConfigurationRecordv1 {
 
 ### Semantics ### {#av1codecconfigurationbox-semantics}
 
-Previous versions of this specification defined v0 of the AV1CodecConfigurationRecord. Use of AV1CodecConfigurationRecord v0 is deprecated.
+Previous versions of this specification defined the AV1CodecConfigurationRecord v0 and AV1CodecConfigurationRecordv1 as extending FullBox. Use of those versions of the AV1CodecConfigurationRecord is deprecated. Files conformant to this specification SHALL use the AV1CodecConfigurationRecord that extends Box.
```

Comment:
Is it even necessary to say this? Is it clear enough?

Reply:
Strictly speaking the records are not extending FullBox or Box. Given that we are breaking the compatibility with the previous draft, I think we can safely remove the entire paragraph.